src/ReportManager/DetailsSection/SchemaForm/fields/Text/index.js

```diff
@@ -50,20 +50,28 @@ const Text = ({ id }) => {
   }, [details.defaultInput, id, onFieldChange, value]);
 
   return <div data-testid={`schema-form-text-field-${id}`}>
-    <label className={styles.label} htmlFor={id}>{label}</label>
+    <label className={`${styles.label} ${hasError ? styles.error : ''}`} htmlFor={id}>{label}</label>
```

Comment:
**Related to AJV validations**
I added error state to the input and the corresponding aria tags to make the field accessible.